ui-core/src/main/java/com/adyen/checkout/components/ui/adapter/ClickableListRecyclerAdapter.kt

```diff
@@ -17,10 +17,10 @@ abstract class ClickableListRecyclerAdapter<ViewHolderT : RecyclerView.ViewHolde
     var onItemCLickedListener: OnItemCLickedListener? = null
 
     @CallSuper
-    override fun onBindViewHolder(viewHolderT: ViewHolderT, position: Int) {
-        viewHolderT.itemView.setOnClickListener {
+    override fun onBindViewHolder(viewHolder: ViewHolderT, position: Int) {
```

Comment:
This `abstract` class is only extended by 1 class. Is this abstraction really needed?

Reply:
Yeah you are right, there are probably other interfaces with the same behavior implemented in other parts of the code too. But I think this change falls outside of the scope of this PR.